modules/dataframe/src/jsondataframeconversion.cpp

```diff
@@ -93,7 +93,7 @@ void addDataFrameColumnHelper(json::value_t valueType, std::string header, DataF
             df.addColumn<uint32_t>(header, 0u);
             break;
         case json::value_t::number_float:  ///< number value (floating-point)
-            df.addColumn<float>(header, 0u);
+            df.addColumn<double>(header, 0u);
```

Comment:
Just not that this change might break cases where we later use the column in a plot for example.

Reply:
Sure, we can also agree on a "common" format for floating points. The `CSVSource`, for example, has an option for double precision, which defaults to single precision.